### ISSUE TITLE
Not compliant with spec

### ISSUE THREAD

**Issue by Quezler** [alefesouza/thanosjs#1]
The [selection ](https://github.com/alefesouza/thanosjs/blob/master/src/thanos.js#L23) s biassed, there should be no exemption amongst the files, which is in direct violation of the [spec](https://en.wikipedia.org/wiki/The_Infinity_Gauntlet), and should thus be fixed, are pull requests accepted? 🤔

**Comment by abhchand** [comment]
I agree. For true randomness, you need to be willing to delete _everything_
